### PR TITLE
style: navbar ui updated

### PR DESCRIPTION
**Pull request by Shubham-Rasal**
## Description

Modified the margin of the navIcon which when hovered was very close to the search bar.

## Screen capture

 Before       

interface-before-web
<br>
After (Desktop)
<br>
interface-after-web 

 

### Automated testing

- [x] Unit test
- [x] Integration/E2E test

### CODE REVIEW

File: src/components/NavBar/NavIcon.css.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const navIcon = style([
     cursor: 'pointer',
     borderRadius: '8',
     transition: '250',
+    margin: '4',
```
Comment (cbachmeier) [inline src/components/NavBar/NavIcon.css.ts:18]: `margin-right` would be better for this purpose. Having the top-margin un-aligns the menu button
Screen Shot 2023-05-23 at 10 26 05

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavIcon.css.ts (hunk 1 above)
Comment (Shubham-Rasal) [reply src/components/NavBar/NavIcon.css.ts:18]: On it

File: src/components/NavBar/NavIcon.css.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ export const navIcon = style([
     cursor: 'pointer',
     borderRadius: '8',
     transition: '250',
+    marginRight: '4',
```
Comment (cbachmeier) [inline src/components/NavBar/NavIcon.css.ts:18]: This changes the style of the generic `NavIcon` component which is also shared with the NFT Shopping bag icon as well as the overflow menu. Updating the NFT Shopping bag is unintended. But you can move this style change one level up to the flex `Box` that wraps this `NavIcon` in `MenuDropdown.tsx`

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavIcon.css.ts (hunk 2 above)
Comment (Shubham-Rasal) [reply src/components/NavBar/NavIcon.css.ts:18]: That should work , will try it.

Comment (Shubham-Rasal) [reply src/components/NavBar/NavIcon.css.ts]: Added the margin to the box.

**Review by cbachmeier — APPROVED** [review]
LGTM 👍

**Comment by Shubham-Rasal** [comment]
Review please @tinaszheng

**Pull request merged**